### PR TITLE
No need to export, build-tools will cache

### PR DESCRIPTION
**Pull request by ari-gold**

### CODE REVIEW

**Comment by ari-gold** [comment]
@greg-1-anderson

**Comment by greg-1-anderson** [comment]
I think it would be good to retain the info about the GITHUB_TOKEN and CIRCLE_TOKEN, perhaps as an optional step.

**Comment by ari-gold** [comment]
IF there are legitimate use cases for the extra steps I'm +1 on keeping, but maybe move down to an FAQ or something to streamline the "quickstart" . @rachelwhitton or @alexfornuto would you be able to pick this up?

**Comment by greg-1-anderson** [comment]
The largest use-case is for folks using version 1.x of the Terminus Build Tools, which does not cache the credentials. I'm going to tag 2.0.0 stable shortly, though.

**Comment by greg-1-anderson** [comment]
Oh nevermind I re-read my reminder above that we are not caching in 1.0, and I haven't tagged 2.0 stable yet. Nevermind. 👍

File: source/_docs/guides/build-tools/02-create-project.md [hunk 1]
```diff
@@ -49,10 +30,29 @@ In this section we'll use the Terminus Build Tools Plugin to create a new Panthe
...
+
+    ```bash
+    export GITHUB_TOKEN=yourGitHubToken
+    export CIRCLE_TOKEN=yourCircleCIToken
+    ```
+
+    If you need to replace a token, navigate to your [project settings page in CircleCI](https://circleci.com/docs/2.0/env-vars/#adding-environment-variables-in-the-app){.external}.
+    </div>
+    </div>
+
```
Comment (rachelwhitton) [inline source/_docs/guides/build-tools/02-create-project.md]: Two drop down panels in row like this makes it hard to tell what we want to user to do

Let's move the token info out of the drop down and place it either 1) top of this page or 2) before you begin on the intro page -- just so we give the info on tokens before the user has to interact with them.

**Pull request merged**